blocks/identity-block/utils/validate-redirect-url.js

```diff
@@ -1,10 +1,18 @@
-const validateURL = (url) => {
+const validateURL = (url, contextPath = '') => {
 	if (!url) return null;
 	const validationRegEx = /^\/[^/].*$/;
 	const valid = validationRegEx.test(url);
+
 	if (valid) {
+		return `${window.location.origin}${contextPath}${url}`;
```

Comment:
This was added to fix the issue with AppSec requirements with out having to do too much with configurations. `contextPath` in local will append `/pf` to the url. We will have to test this out on the remote server because I am not sure what that value is on the remote server.